extensions/wikia/ArticleVideo/styles/article-video.scss

```diff
@@ -0,0 +1,250 @@
+	background-size: cover;
+	margin-bottom: 11px;
+	padding-top: 56.25%; // we want to have 16:9 ratio div size
+	position: relative;
+	width: 100%;
+
+	.video-details {
+		* {
+			opacity: 0.6;
+		}
+	}
+
+	.video-thumbnail {
+		bottom: 0;
+		left: 0;
+		position: absolute;
+		right: 0;
+		top: 0;
+		width: 100%;
+	}
+
+	&:before {
+		background-image: $video-overlay-gradient;
+		bottom: 0;
+		content: '';
+		left: 0;
+		position: absolute;
+		right: 0;
+		top: 50%;
+		z-index: 1;
+	}
+
+	.video-title {
+		bottom: 41px;
+		color: #fff;
+		font-size: 24px;
+		left: 38px;
+		position: absolute;
+	}
+
+	.video-time {
+		bottom: 73px;
+		color: #656e78;
+		font-size: 14px;
+		font-weight: bold;
+		left: 38px;
+		position: absolute;
+		text-transform: uppercase;
+	}
+
+	.video-play-button {
+		bottom: 15px;
+		height: 120px;
+		position: absolute;
+		right: 15px;
+		width: 120px;
+	}
+
+	.video-container {
+		bottom: 0;
+		left: 0;
+		position: absolute;
+		right: 0;
+		top: 0;
+		transform: translate3d(0, 0, 0); // hack for iOS for changing position to fixed while scrolling
+		z-index: 2;
+
+		.video-thumbnail {
+			display: none;
+			position: static;
+		}
+
+		.close {
+			display: none;
+			left: 0;
+			padding: 7px;
+			position: absolute;
+			top: 0;
+			z-index: 1;
+		}
+	}
+
+	&.ready-to-play {
+		cursor: pointer;
+
+		.video-container {
+			.video-details {
+				* {
+					opacity: 1;
+				}
+			}
+		}
+
+		.spinner {
+			display: none;
+		}
+	}
+
+	&.playing.ready-to-play {
+		cursor: default;
+	}
+
+	&.collapsed-ready {
+		.video-container {
+			left: auto;
+			position: fixed;
+			top: auto;
+
+			.video-thumbnail {
+				display: block;
+			}
+		}
+	}
+
+	&.collapsed {
+		.video-container {
+			bottom: 45px !important;
+			left: auto;
+			position: fixed;
+			right: 20px !important;
+			top: auto;
+			transition: right $video-collapse-animation-speed, bottom $video-collapse-animation-speed, width $video-collapse-animation-speed;
+			width: 225px !important;
+
+			.close {
+				cursor: pointer;
+				display: block;
+				z-index: 10001;
+			}
+
+			.video-thumbnail {
+				display: block;
+				height: $video-collapsed-height !important;
+				transition: height $video-collapse-animation-speed;
+			}
+		}
+
+		.video-details {
+			align-items: center;
+			background: #000;
+			display: flex;
+
+			* {
+				opacity: 0.7;
+			}
+
+			.video-details-left {
+				flex: 1;
+				margin-left: 10px;
+			}
+
+			.video-time {
+				font-size: 10px;
+				position: static;
+			}
+
+			.video-title {
+				font-size: 12px;
+				position: static;
+			}
+
+			.video-play-button {
+				height: 60px;
+				margin: 0;
+				position: static;
+				width: 60px;
+			}
+		}
+
+		.ooyala-article-video {
+			bottom: auto;
+			height: $video-collapsed-height !important;
+			transition: height $video-collapse-animation-speed;
+		}
+
+		.spinner {
+			top: $video-collapsed-height / 2;
+		}
+	}
+
+	&.animating {
+		.oo_controls {
+			display: none;
+		}
+	}
+
+	.spinner {
```

Comment:
Is it a new spinner that we're going to start using site-wide? If yes we should consider moving it's default styles to DesignSystem.